tests/Unit/Instrumentation/AwsSdk/AwsSdkInstrumentationTest.php

```diff
@@ -37,6 +55,55 @@ public function testInstrumentationInit()
         );
     }
 
+    public function testSetXrayPropagator()
```

Comment:
this doesn't test what it says - it should probably add a propagator to the instrumentation class, get it back, and `$this->assertSame()` the input and output